tasks/install_packages.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: Install ovirt-engine/rhvm package
+- name: Install ovirt-engine/rhvm/rhevm package
   package:
     name: "{{ ovirt_engine_setup_product_type }}"
```

Comment:
Wouldn't it be better from user perspective to place rhvm or rhevm according to specified version? 4.1 - rhevm, anything else is rhvm

Reply:
You 're right, there is not need of parameter `ovirt_engine_setup_product_type`. I will take care to use the correct package name by checking the version.
Thanks

Reply:
Well, you need to somehow distinguish between oVirt and RHV, right?

Reply:
Right, so I guess then we 'll change this `ovirt_engine_setup_product_type` variable to actually depict if we are handling downstream or upstream. So it can have 2 values [''downstream", "upstream"]
Then knowing the version and the product type I can decide regarding the correct alias that should be used for product installation.
This change is breaking backward compatibility though, and we 'll need to make major release if we include this fix.
WDYT? Or you have something else in mind?

Reply:
I'd prefer ["oVirt", "RHV"], case insensitive of course :-)

I think we have skipped releasing the role in 4.2.1, so 4.2.2 should be the 1st official release. So we should be OK around backward compatibility